files/ko/web/svg/attribute/cx/index.md

```diff
@@ -0,0 +1,147 @@
+---
+title: cx
+slug: Web/SVG/Attribute/cx
+translation_of: Web/SVG/Attribute/cx
+---
+{{SVGRef}}
+
+**`cx`** 속성은 중심점의 x축 좌표를 정의합니다.
+
+다음 SVG 요소와 함께 이 속성을 사용할 수 있습니다:
+
+- {{SVGElement("circle")}}
+- {{SVGElement("ellipse")}}
+- {{SVGElement("radialGradient")}}
+
+## Example
+
+```css hidden
+html,body,svg { height:100% }
+```
+
+```html
+<svg viewBox="0 0 300 100" xmlns="http://www.w3.org/2000/svg">
+  <radialGradient cx="25%" id="myGradient">
+    <stop offset="0"    stop-color="white" />
+    <stop offset="100%" stop-color="black" />
+  </radialGradient>
+
+  <circle cx="50" cy="50" r="45"/>
+  <ellipse cx="150" cy="50" rx="45" ry="25" />
+  <rect x="205" y="5" width="90" height="90" fill="url(#myGradient)" />
+</svg>
+```
+
+{{EmbedLiveSample("Example", 100, 100)}}
+
+## circle
+
+{{SVGElement('circle')}} 의 경우, `cx`는 모양 중심의 x축 좌표를 정의합니다.
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">Value</th>
+      <td>{{cssxref("length-percentage")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">Default value</th>
+      <td><code>0</code></td>
+    </tr>
+    <tr>
+      <th scope="row">Animatable</th>
+      <td>Yes</td>
+    </tr>
+  </tbody>
+</table>
+
+> **Note:** SVG2 `cx`로 시작하는 것은 *기하학적 속성* 입니다. 즉, 이 속성은 circle의 CSS 속성으로도 사용할 수 있습니다.
+
+## ellipse
+
+{{SVGElement('ellipse')}}의 경우, `cx`는 모양 중심의 x축 좌표를 정의합니다.
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">Value</th>
+      <td>{{cssxref("length-percentage")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">Default value</th>
+      <td><code>0</code></td>
+    </tr>
+    <tr>
+      <th scope="row">Animatable</th>
+      <td>Yes</td>
+    </tr>
+  </tbody>
+</table>
+
+> **Note:** SVG2 `cx`로 시작하는 것은 *기하학적 속성* 입니다. 이는 이 속성이 ellipse의 CSS 속성으로도 사용될 수 있음을 의미합니다.
+
+## radialGradient
+
+{{SVGElement('radialGradient')}}의 경우 `cx`는 radial Gradient에 대한 끝 원의 x축 좌표를 정의합니다.
+
+<table class="properties">
+  <tbody>
+    <tr>
+      <th scope="row">Value</th>
+      <td>{{cssxref("length")}}</td>
+    </tr>
+    <tr>
+      <th scope="row">Default value</th>
+      <td><code>50%</code></td>
+    </tr>
+    <tr>
+      <th scope="row">Animatable</th>
+      <td>Yes</td>
+    </tr>
+  </tbody>
+</table>
+
+## 예제
```

Comment:
The preview didn't work because the example was declared as h2. In the actual content repo, it was declared as h4. understand. thank you for the reply. 🙇🙇